### PR TITLE
ENH: simplified linearized deformations implementation and test

### PR DESCRIPTION
**Pull request by adler-j**
* Removed code that duplicates functionality from other parts of ODL
* Removed tests for the above (since it is already tested)
* Changed test function form hat to Gaussian, reduced test error bars by factor 10.

### CODE REVIEW

File: odl/deform/linearized.py [hunk 1]
```diff
@@ -132,167 +130,26 @@ def derivative(self, displacement):
...
-        -------
-        adjoint : `LinDeformFixedTemplDerivAdj`
-            The adjoint of the operator.
-        """
-        return LinDeformFixedTemplDerivAdj(self.template, self.displacement,
-                                           self.def_grad)
-
-
-class LinDeformFixedTemplDerivAdj(Operator):
-
```
Comment (chongchenmath) [inline odl/deform/linearized.py]: The adjoint is removed?

Comment (adler-j) [inline odl/deform/linearized.py]: Adjoint is provided by the `PointwiseInner` class.

**Comment by adler-j** [comment]
This has been rebased to master, which the upstream branch has not. I suggest we get this merged to that branch and get a final review done.

**Comment by kohr-h** [comment]
Yes, do that.

**Pull request closed without merging**